orion_mcp.py

```
@@ -118,287 +170,401 @@ def get_data_source_resource() -> str:
+                }
+            }
+        },
+    }
+
+    if workload:
+        query["query"]["bool"].setdefault("filter", []).append(
+            {"wildcard": {"upstreamJob.keyword": {"value": f"*{workload}*"}}}
+        )
+
+    try:
+        async with httpx.AsyncClient(timeout=30, verify=False) as client:
```

Ruff (0.16.1)

[error] Probable use of `httpx` call with `verify=False` disabling SSL certificate checks

(S501)